sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -355,6 +358,7 @@ private QueryObservable createQuery(Func1<Set<String>, Boolean> tableFilter, Str
         .map(query) // DatabaseQuery maps to itself to save an allocation.
         .onBackpressureLatest() // Guard against uncontrollable frequency of upstream emissions.
         .startWith(query) //
+        .compose(queryTransformer) // Apply the user's query transformer
```

Comment:
I thought in the `.takeUntil()` case, it would be beneficial to unsubscribe before any final event made the jump from the subscribeOn scheduler to the observeOn scheduler. Probably not a huge deal though. Is there a negative to putting it before?